lib/open_api/serializer.rb

```diff
@@ -8,6 +8,8 @@ def as_json(arg = {})
           hash[attr_key] = hash[attr_key].iso8601 if hash[attr_key].kind_of?(Time)
           hash[attr_key] = hash[attr_key].to_s if attr_key.ends_with?("_id") || attr_key == "id"
         end
+
+        hash.delete("context")
```

Comment:
I feel it's better to use `:except` options of as_json to exclude `context`, like:
```
as_json(except: :context)
```

Reply:
Ok, updated to use `except:` syntax in the `Response` concern